providers/ns1/ns1Provider.go

```diff
@@ -88,6 +89,52 @@ func (n *nsone) GetZoneRecords(domain string) (models.Records, error) {
 	}
 	return found, nil
 }
+// Get DNSSEC status for zone. Returns true for enabled, false for disabled
```

Comment:
Go style: Start the comment with the name of the function.
```suggestion
// GetZoneDNSSEC gets the DNSSEC status for zone. Returns true for enabled, false for disabled
```

Reply:
ah yeah, always forget these. Thanks!